Makefile.dep

```diff
@@ -31,6 +31,17 @@ ifneq (,$(filter gnrc_uhcpc,$(USEMODULE)))
     USEMODULE += fmt
 endif
 
+ifneq (,$(filter nordic_softdevice_ble,$(USEPKG)))
+    USEMODULE += softdevice_handler
+    USEMODULE += ble_common
+    USEMODULE += ble_6lowpan
```

Comment:
Hmm, that file (ble_6lowpan.a) is a blob from Nordic. I could rename the file, but it would be some lines of makefile logic just hiding that fact.